src/Drawer/Drawer.js

```diff
@@ -41,7 +41,8 @@ export const styleSheet = createStyleSheet('MuiDrawer', (theme) => {
       WebkitOverflowScrolling: 'touch', // Add iOS momentum scrolling.
     },
     docked: {
-      flex: '0 0 auto',
```

Comment:
That's the most challenging change I have seen.
It's quite different.
On Inbox by Google, it seems that a Drawer can be  docked and open/closed without messing with the body width. That change gives us the flexibility to do the same.